src/app/app-routing.module.ts

```diff
@@ -1,15 +1,17 @@
+import { LoginPageComponent } from './auth/login-page/login-page.component';
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
 import { LandingPageComponent } from './landing-page/landing-page.component';
 import { QuizComponent } from './quiz/quiz.component';
 
 const routes: Routes = [
-  {path: '', component : LandingPageComponent},
-  {path: 'api', component : QuizComponent}
+  { path: '', component: LandingPageComponent },
+  { path: 'api', component: QuizComponent },
+  { path: 'login', component: LoginPageComponent },
 ];
 
 @NgModule({
   imports: [RouterModule.forRoot(routes)],
-  exports: [RouterModule]
+  exports: [RouterModule],
 })
-export class AppRoutingModule { }
+export class AppRoutingModule {}
```

Comment:
Why are we changing spacing with each MR? Perhaps there is some convention that we should use?
` { path: '', component: LandingPageComponent }` here I see you are leaving spaces between brackets so it creates inconsistencies

Reply:
Lukas is using VSCode, I use Webstorm. Personal preference. Sometimes linter does these things, but id doesn't affect the neither performance, neither readability. I see no problem here.

Reply:
In this case I suggest setting up your IDEs to both use teh same formatting rules. This helps to keep code clean and consistent.

Reply:
Not worth the time investment.

Reply:
I wouldnt say so. Keep consistent code culture IS important. Also, it reduces your MR size as you dont have to keep commiting same changes all over.

Reply:
Project is coming to an end and I would much rather spend time on coding rather than setting up the IDEs which should've been done at the beginning of project. Also it's not that hard to ignore unimportant changes in the MR, is it?

Reply:
Nemanau kad tai vertins.